content/pages/meetings/2018-03-12_fr.md

```diff
@@ -0,0 +1,56 @@
+Date: 2018-03-18
+Modified: 2018-03-18
+Authors: Michael Kalbermatten
+URL: meetings/2018-03-12
+Save_as: meetings/2018-03-12.html
+Slug: meetings/2018-03-12
+Lang: fr
+Status: hidden
+
+<br />
+La plupart des points sont repris dans la présentation de Camptocamp, [gmfusrgrp_20180312.pdf]({filename}/documents/meetings/2018-03-12/gmfusrgrp_20180312.pdf) 
+
+Ci-dessous se trouvent quelques compléments concernant l'un ou l'autre point.
+
+### Version 2.3 - Release
+
+Le premier release concernant la version 2.3 aura lieu fin mars.
+
+### Développements futurs
+
+* Vector tiles : swisstopo a commencé l'intégration des VT dans leur infrastructure. Le groupe GMF devrait suivre cela, éventuellement aussi participer à ce développement et prévoir une intégration dans GMF comme nouvelle source de données. Le coordinateur du groupe va contacter l'équipe de geo.admin afin que le groupe puisse élaborer une stratégie en accord avec geo.admin.
+
+* Docker : les futures versions (à partir de la version 2.3) pourront être buildées et déployées via Docker (qui va devenir une dépendance obligatoire). La groupe n'a pour l'instant que peu, voire pas, de connaissance concernant Docker. Le coordinateur du groupe et Camptocamp vont donc mettre sur pieds une formation Docker spécialement orientée GMF. Celle-ci aura probablement lieu en mai-juin 2018. Plus d'information va suivre.
+
+### Product management
+
+Des problèmes de support ont apparu lors de la version 2.2. Le suivi et la coordination des release entre le groupe GMF et Camptocamp n'est pas optimal.
+
+Il y a plusieurs pistes et le groupe GMF a mis sur pieds un groupe de réflexion qui devra se pencher sur la problématique du suivi. Le groupe se compose de:
+
+* Camptocamp
+* Cartoriviera
+* Kanton Schwytz
+* Canton Ticino
+
+L'output du groupe devrait permettre de mettre en place des solutions de suivi du release, voir un changement de paradigme concernant la gestion des release (Scrum ?).
+
+Un sondage va encore être proposé au membre du groupe concernant l'acceptation des release.
+
+### Financement GMF 2.4
+
+L'offre de Camptocamp a été envoyée aux partenaires finançant la version 2.4. Ils ont jusqu'au mercredi 21 mars 2018 pour donner un retour au coordinateur du PSC.
+
+### Profil LIDAR
+
+O. Monod a présenté le développement du profil LIDAR. Ce dernier est en cours d'intégration dans NGEO. La présentation est disponible ici : [GMF-lidar-profile_20180312.pdf]({filename}/documents/meetings/2018-03-12/GMF-lidar-profile_20180312.pdf)
+
+### Texte décrivant la communauté
+
+L. Teba (Prilly) et G. Sollberger (géocommunes) ont élaboré un texte qui décrit la communauté GMF. Ce texte a été soumis au groupe et aucune remarque n'a été faite concernant ce dernier. Il va donc être utilisé pour remplacer la page de garde du site Geomapfish. Une refonte du site va être proposée par le coordinateur.
+
+### Agenda
+
+* Prochaine séance
+    * Yverdon-les-Bains, 03 septembre 2018 de 13h15-16h00
```

Comment:
It wasn't the 27 August?

Reply:
Nope, @gnerred just confirmed to me that it was on the 3rd september...